src/plugins/controlbox/utils.js

```diff
@@ -26,9 +26,12 @@ export function disconnect () {
      * we reconnect, "onConnected" will be called,
      * to fetch the roster again and to send out a presence stanza.
      */
-    const view = _converse.state.chatboxviews.get('controlbox');
-    view.model.set({ 'connected': false });
-    return view;
+    try{
+        const view = _converse.state.chatboxviews.get('controlbox');
+        view.model.set({ 'connected': false });
```

Comment:
What is the error/exception that you're seeing?

If it's due to `view` not being defined, then you can do:
```
view?.model.set({ connected: false });
```